File: gabriel/Problema0378/Problema0378.java
package Problema0378;
import java.util.Scanner;

public class Problema0378 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        int matriz [][] = new int [4][3];
        float soma = 0, contador = 0, media;
        
        for(int i=0; i<4; i++){
            for(int j=0; j<3; j++){
                System.out.println("Digite o numero armazenado na linha " +i+ " e da couna "+j);
                matriz [i][j] = Integer.valueOf(input.nextLine());
            }
        }
        
        for(int i=0; i<4; i++){
            for(int j=0; j<3; j++){
                if(i % 2 == 0){
                    soma = soma + matriz [i][j];
                    contador = contador + 1;
                }
            }
        }
        
        media = soma / contador;
        
        System.out.println("A média dos elementos da linhas pares: "+media);
    }
    
}

File: gabriel/Problema0378/Problema0378.js
var matriz = [[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0]];
var soma = 0, contador = 0, media;

for(var i=0; i<4; i++){
    for(var j=0; j<3; j++){
        matriz [i][j] = parseFloat(prompt("Digite o numero armazenado na linha " +i+ " e da couna "+j));
    }
}

for(var i=0; i<4; i++){
    for(var j=0; j<3; j++){
        if(i % 2 == 0){
            soma = soma + matriz [i][j];
            contador = contador + 1;
        }
    }
}

media = soma / contador;

console.log("A média dos elementos nas linhas pares: "+media);
    